frontend/src/components/common/radio/radio.tsx

```diff
@@ -13,31 +15,34 @@ interface Props {
   onChange: InputChangeCallback;
```

Comment:
```suggestion
```
We don't need it if we use `useForm`;

Delete `InputChangeCallback ` is we don't use it

Add `errors` props 

Remove `hasError` prop